config/watcher_local_test/kustomization.yaml

```diff
@@ -3,24 +3,26 @@
 # that we do not want in the default configuration (e.g. vendor-specific gateway implementation)
 # but are necessary for testing the watcher on locally provisioned clusters using k3d.
 resources:
-- ../manager
+- ../default
 
 # [WATCHER] To enable the watcher, uncomment all the sections with [WATCHER]
 # [ISTIO] To enable istio, uncomment all sections with 'ISTIO'.
 components:
-  - ../crd
-  - ../rbac
   - ../watcher
   - ../istio
 
-namespace: kcp-system
 
 generatorOptions:
   disableNameSuffixHash: true
 
+# TODO REMOVE this nameprefix and adapt whole watcher setup -> everything should have `lifecycle-manager-` as prefix
```

Comment:
Can we apply this TODO in this PR?

Reply:
Ups,...leftover TODO - as you can see this was already done